src/lib/ffi/ffi.h

```diff
@@ -89,7 +89,7 @@ int botan_make_error(const char* msg, const char* func, int line);
 
 // This would seem to require both saving the message permanently
 catch(std::exception& e) {
-return botan_make_error_from_transient_string(e.what(), BOTAN_FUNCTION, __LINE__);
+return botan_make_error_fromransient_string(e.what(), BOTAN_FUNCTION, __LINE__);
```

Comment:
Oh passed CI because this code is just part of a comment sketching out future work to the API.